src/js/layerlist/LayerList.js

```diff
@@ -176,10 +176,30 @@ L.Dropchop.LayerList = L.Control.extend({
     */
     _handleLayerChange: function(obj, e){
         var inputEl = e.target;
+        var selected = obj.domElement.getElementsByClassName('layer-name')[0].classList.contains('selected')
         if (inputEl.checked) {
-            this._map.addLayer(obj.layer);
+            // If changed element in selection, alter entire selection
+            if (selected) {
+                this.selection.list.map(function(obj){
```

Comment:
I'm starting to think that basic `for` loops are better than `.map`.  Faster, possibly more intuitive to reader, no need for `.bind`.